workflow-templates/upgrade-python-requirements.properties.json

```
@@ -0,0 +1,11 @@
+{
+    "name": "Python Requirements Upgrade Workflow",
+    "description": "Requirements Upgrade workflow template for edX",
+    "iconName": "upgrade-python-requirements-icon",
+    "categories": [
+        "python"
+    ],
+    "filePatterns": [
+        "setup.py$"
```

```suggestion
        "setup.py$",
        "Makefile"
```
Since this workflow runs `make upgrade`